pytest-embedded/pytest_embedded/utils.py

```diff
@@ -80,6 +81,13 @@ def find_by_suffix(suffix: str, path: str) -> List[str]:
     return res
 
 
+def prettify_xml(file_path: str):
```

Comment:
this function will never be reused... how about we move it to the `plugin.py` instead?

Reply:
Sure, I've moved the function to the correct place.